### PR TITLE
more over light effects. I should really make this into a proc.

### PR DESCRIPTION
**Pull request by chaoko99**
🆑 chaoko99
rscadd: Emissive door bolt, locker locks, emag sparks, vitals monitor indicators.
tweak: made the vitals monitor dense trust me the alternative looked weird as hell.
tweak: slimmed down some of the memory usage in locker generation. 
🆑 
image
image

Not gun lockers though, the fucking snowflakes.

### CODE REVIEW

File: code/game/machinery/vitals_monitor.dm [hunk 1]
```diff
@@ -9,6 +9,7 @@
 	idle_power_usage = 10
 	active_power_usage = 100
 	stat_immune = NOINPUT
+	density = TRUE
```
Comment (Roland410) [inline code/game/machinery/vitals_monitor.dm]: This probably should have been dense in the first place anyways. Got a pic though?

Comment (chaoko99) [reply code/game/machinery/vitals_monitor.dm]: Unfortunately not. Basically it just hid half your sprite if you stood on it, and most of it  if you were lying down.

**Comment by Roland410** [comment]
By the way, the build errored out on Travis. You'll want to force push something or close and reopen the PR.

**Comment by chaoko99** [comment]
give me an hour or so before merging this, I'm adding another feature.

**Comment by chaoko99** [comment]
image
there, good to go.

**Comment by chaoko99** [comment]
rerunning travis as that's not my fail.

**Comment by chaoko99** [comment]
ready to be merged.

**Pull request merged**